src/hijackers/dynamicScript.ts

```diff
@@ -0,0 +1,26 @@
+import { execScripts } from 'import-html-entry';
+import { Freer } from '../interfaces';
+
+export default function hijack(proxy: Window): Freer {
+  const rawHtmlAppendChild = HTMLHeadElement.prototype.appendChild;
```

Comment:
move the rawHtmlAppendChild to the top of module, see https://github.com/umijs/qiankun/blob/master/src/hijackers/dynamicStylesheet.ts#L8